edenai_apis/apis/microsoft/microsoft_llm_api.py

```
@@ -0,0 +1,85 @@
+from typing import List, Type, Union, Optional
+
+import httpx
+from openai import BaseModel
+from edenai_apis.features.llm.llm_interface import LlmInterface
+from edenai_apis.features.llm.chat.chat_dataclass import ChatDataClass
+
+
+class MicrosoftLLMApi(LlmInterface):
+    def llm__chat(
+        self,
+        messages: List = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)